src/scripts/main.js

```diff
@@ -354,7 +354,33 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function calculateAge(born, died) {
+  return died - born;
+}
+
+function calculateCent(year) {
+  return Math.ceil(year / 100);
```

Comment:
The `calculateCent` function uses `Math.ceil`, which might not give the correct century for years exactly at the start of a century (e.g., 1900). Consider using `Math.floor` to ensure the correct century is calculated.